composer.json

```diff
@@ -9,7 +9,7 @@
 	},
 	"require": {
 		"php": "^7.0",
-		"squizlabs/php_codesniffer": "~2.8.1"
+		"squizlabs/php_codesniffer": "dev-master"
```

Comment:
Yes, I'll use ^3.0.1 :) I just want to test that there is no other bug in PHPCS.